control/changes.go

```diff
@@ -120,7 +121,7 @@ func ParseChangesFile(path string) (ret *Changes, err error) {
 // to something invalid if you're not using those functions.
 func ParseChanges(reader *bufio.Reader, path string) (*Changes, error) {
 	ret := &Changes{Filename: path}
-	return ret, Unmarshal(ret, reader)
+	return ret, rfc2822.Unmarshal(ret, reader)
```

Comment:
So, Unmarshal needs a bit of an API cleanup. Can we leave that in control as-is, with a deprecation warning, and create a better Unmarshal / Marshal?

In particular, I'd need to clean up the keyring support and also the argument order.